holidays/countries/jamaica.py

```diff
@@ -12,72 +12,74 @@
 from datetime import date
 from datetime import timedelta as td
 
-from dateutil.easter import easter
-
 from holidays.calendars import _get_nth_weekday_from, _get_nth_weekday_of_month
-from holidays.constants import JAN, MAY, AUG, OCT, DEC, MON
+from holidays.constants import MAY, AUG, OCT, MON
 from holidays.holiday_base import HolidayBase
+from holidays.holiday_groups import ChristianHolidays, InternationalHolidays
 
 
-class Jamaica(HolidayBase):
+class Jamaica(HolidayBase, ChristianHolidays, InternationalHolidays):
     """
     https://en.wikipedia.org/wiki/Public_holidays_in_Jamaica
     https://www.mlss.gov.jm/wp-content/uploads/2017/11/The-Holidays-Public-General-Act.pdf
     """
 
     country = "JM"
 
+    def __init__(self, *args, **kwargs) -> None:
+        ChristianHolidays.__init__(self)
+        InternationalHolidays.__init__(self)
+        super().__init__(*args, **kwargs)
+
     def _populate(self, year):
-        def _add_with_observed(hol_date: date, hol_name: str) -> None:
-            self[hol_date] = hol_name
+        def _add_observed(hol_date: date) -> None:
```

Comment:
I guess we should use some consistent approach for all the "extra" stuff. 
Are you okay with moving this above the `_populate()`?

Reply:
Now I've made it a separate (and more flexible) method.